### PR TITLE
Update Quest wiki

### PR DESCRIPTION
**Pull request by Corantin**
Make the quests wiki more descriptive to help users to navigate in the app

### CODE REVIEW

File: projects/quests.md [hunk 1]
```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
...
+This page is the detailed version of the quest showing more information :
+
+- The claim deposit (amount locked during the claiming process)
+- The full description (not summarized)
+- The list of the **Players**' claims
+- The action buttons (needs to be connected)
+  - Fund (where Patrons can fund the quest)
+  - Schedule claim (where Players can submit their evidence of task completion)
+
+![](https://i.imgur.com/HlJA7fs.png)
```
Comment (kamikazebr) [inline projects/quests.md]: Maybe upload the necessary files to the repo, so we keep versioning it and not relay on external sources, wdty?

File: projects/quests.md [hunk 2]
```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
 
 The evidence required for completing a quest will depend on the work expected. For a quest on writing a blog post, the evidence required will likely include a link to the blog post. It may also specify the minimum length of the blog post and that it hasn't been copied from somewhere else. For a quest on adding a feature to a web UI, the evidence required will likely include a link to the code written, some screenshots of the deployed feature and some reference to the code standard expected.
 
+## Pages
+
+### List view
+
+![](https://i.imgur.com/hRtXi66.png)
```
Comment (kamikazebr) [inline projects/quests.md]: Here also

File: projects/quests.md [hunk 3]
```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
...
+
+This page consists of a small dashboard and a list of summarized Quests.
+
+The dasboard shows the following information:
+
+- Total of the active Quests bounties in USD
```
Comment (kamikazebr) [inline projects/quests.md:43]: maybe make some * or note about the some tokens dont show the USD value?

File: projects/quests.md [hunk 4]
```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
...
+  - Fund (where Patrons can fund the quest)
+  - Schedule claim (where Players can submit their evidence of task completion)
+
+![](https://i.imgur.com/HlJA7fs.png)
+
+## Create a quest
+
+**Creators** can create a Quest only when connected to a wallet. The **Create quest** buttons will then appear in the header/dashboard/footer.
+
+The creator must submit the following in 2 steps :
```
Comment (kamikazebr) [inline projects/quests.md:72]: Maybe correct it to 3 steps?
```suggestion
The creator must submit the following in 3 steps :
```

File: projects/quests.md [hunk 5]
```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
...
+   - Initial bounty
+   - Expire time (after that, Players will not be able to schedule new claims)
+   - Fallback address (the address where the funds can be restored once the quest expires)
+3. The Creator needs to deposit a collateral that will be locked until the Quest expiration date. He will need to Reclaim the deposit to recover it
+
+## Fund a quest
+
+**Patrons** can fund a Quest only when connected to a wallet. The Fund button is in the detail view of a Quest.
+
+The Patron should only submit the amount of funding for the Quest reward token.
```
Comment (kamikazebr) [inline projects/quests.md]: Its not clear, what it means?

File: projects/quests.md [hunk 6]
```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
...
+
+## Claim a Quest bounty
+
+**Players** can claim the bounty only when connected to a wallet. The **Schedule claim** button is in the detailed view of a Quest.
```
Comment (kamikazebr) [inline projects/quests.md]:
```suggestion
**Players** can claim the bounty only when connected to a wallet. 
The **Schedule claim** button is in the detailed view of a Quest.
```

Something as new line, i didnt check it in markdown view

File: projects/quests.md [hunk 7]
```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
...
+
+The submitted claim will be delayed for a specific period where other users can challenge the claim if the requirements of the Quest aren't met (see [verification of Quest](quests.md#verifying-a-quest)).
+
+The Player msut submit the following in 2 steps :
```
Comment (kamikazebr) [inline projects/quests.md:99]:
```suggestion
The Player msut submit the following in 3 steps :
```

Here also?

File: projects/quests.md [hunk 8]
```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
...
+
+1. First:
+   - Evidence of completion (should meet the Quest description requirements)
+2. Second:
+   - The UI offers to claim all the Quest's bounty or just a specific amount (Player should pick what is specified in the Quest description)
+   - Player's wallet address to which the reward should be sent when a claim is completed
+3. The Player should deposit a collateral that will be restored once the claim is executed successfully.
```
Comment (kamikazebr) [inline projects/quests.md]: If not successfully, what happen?

File: projects/quests.md [hunk 9]
```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
...
+
+When the delay will be over, the Player can execute the claim by clicking on the **Claim** button next to the corresponding claim. (Note that anyone can execute the claim if the Quest isn't expired)
+
+## Challenge a claim
+
+Anyone can challenge a **Player**'s claim when connected to a wallet. (see [verification of Quest](quests.md#verifying-a-quest))
```
Comment (kamikazebr) [inline projects/quests.md:113]: Liked that inner links, helpful to understand the context

File: projects/quests.md [hunk 10]
```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
...
+The **Challenge** button is next to the Player claim.
+
+A challenger can only challenge during the delay period.
```
Comment (kamikazebr) [inline projects/quests.md]: May say what it is? How much time is that?

File: projects/quests.md [hunk 11]
```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
...
+
+The **Challenger** must submit the following :
+
+1. Reason why this claim should be challenged
+2. A deposit that will be restored when the challenge is resolved in favor of the Challenger
+
+This will then create a dispute in [Celeste](https://celeste.1hive.org/#/dashboard) (Note that with Rinkeby network, a dispute should be resolved manually by contacting us in the #Quests channel in the [1hive discord](https://discord.gg/NaCDVBs2))
```
Comment (kamikazebr) [inline projects/quests.md:123]: Awesome!

File: projects/quests.md (hunk 3 above)
Comment (Corantin) [reply projects/quests.md:43]: Do you mean that I should list the stables here ?

File: projects/quests.md [hunk 12]
```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
...
+
+Once the dispute is resolved in Celeste, a stackholder of the dispute needs to click on the **Resolve** button next to the claim to spread the challenge's result to the app.
+
+If the dispute is resolved ... :
+
+- in favor of the **Player** : The claim will immediatly be executed and **Player** wins the **Challenger** deposit
+- in favor of the **Challenger** : The claim is rejected and the **Challenger** wins the **Player** deposit
```
Comment (kamikazebr) [inline projects/quests.md]: What do you think? Less repetition, the brain can catch it up more fast
```suggestion
If the dispute is resolved in favor of the ... :

- **Player** : The claim will immediatly be executed and **Player** wins the **Challenger** deposit
- **Challenger** : The claim is rejected and the **Challenger** wins the **Player** deposit
```

File: projects/quests.md (hunk 4 above)
Comment (Corantin) [reply projects/quests.md:72]: Its actually 2 steps the third one is just a note but maybe I should not append it to the bullet point

File: projects/quests.md [hunk 13]
```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
...
+
+## Reclaim
+
+The **Creator** can reclaim remaining funds and deposit them when connected to the corresponding wallet. This **Reclaim** button is only in the detailed view of a Quest once the Quest is in the **Expired** state.
+
+If there are pending claims after the expiration date, the **Creator** should't reclaim the funds unless he doesn't want the claiming Players to collect their reward. The creator can execute the claims by himself once the Quest is expired.
```
Comment (kamikazebr) [inline projects/quests.md]: What is Pending claims?

File: projects/quests.md (hunk 5 above)
Comment (Corantin) [reply projects/quests.md]: image

File: projects/quests.md (hunk 8 above)
Comment (Corantin) [reply projects/quests.md]: Its explain somewhere else but an unsuccessful claim is an accepted challenge but yeah I think its more confusing, I'll remove the susscesffully part

File: projects/quests.md (hunk 10 above)
Comment (Corantin) [reply projects/quests.md]: It depends on if Rinkeby and Xdai

Comment (Corantin) [reply projects/quests.md]: But I put 7 days here so 
image

File: projects/quests.md (hunk 13 above)
Comment (Corantin) [reply projects/quests.md]: Claims that has not been rejected or executed (delay or waiting to execute)

**Review by kamikazebr — APPROVED** [review]
Awesome!

**Pull request merged**